### ISSUE TITLE
Auto-Complete: Optimize order of method overloads

### ISSUE THREAD

**Issue by AlCalzone** [ioBroker/ioBroker.javascript#257]
By reordering the method overloads, we can control how the autocomplete works and which overloads are shown first. This way we can first provide help for the most used overloads and then the other ones.

I will try that out at some point, just leaving this issue here so I don't forget.

**Comment by GermanBluefox** [comment]
Done?

**Comment by AlCalzone** [comment]
Not really. This is not very high priority and I'll have to check out some of the other methods too.
If you notice any methods where the most relevant overload is not first, let me know.
